mailpoet/views/newsletters.html

```diff
@@ -62,6 +63,10 @@
       var mailpoet_newsletters_count = <%= newsletters_count %>;
       var mailpoet_send_transactional_emails = <%= json_encode(settings.send_transactional_emails == '1') %>;
       var mailpoet_settings = <%= json_encode(settings) %>;
+      var mailpoet_has_premium_support = <%= has_premium_support ? 'true' : 'false' %>;
```

Comment:
I can see you duplicate all of those variables and translations in a couple of templates. Would it make sense to move them to layout and have them available everywhere?

Reply:
Yes, Thank you for this idea @pavel-mailpoet 👏🏾.
I had been looking for an easy and maintainable way to share the variables but I totally missed this line `<% extends 'layout.html' %>`.
For security reasons, I don't think I should include all of these variables in the layout.html file where it may/may not be publicly accessible (outside wp-admin)
So, I will create a new view file and extend that